<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GmotorTesterSC</class>
 <widget class="QMainWindow" name="GmotorTesterSC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>737</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1259</width>
    <height>737</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GmotorTesterSC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MotorTesting.qrc">
    <normaloff>:/icon/icon/Armor.png</normaloff>:/icon/icon/Armor.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabControlMotor">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>5000</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableMotor">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stop Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Remaining Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Run time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>temperature</string>
       </property>
      </column>
      <item row="0" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="1" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="2" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="3" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="4" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="5" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="6" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="7" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="8" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="9" column="6">
       <property name="text">
        <string>0</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
     </widget>
    </item>
   </layout>
   <zorder>tableMotor</zorder>
   <zorder>tabControlMotor</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionDelete"/>
    <addaction name="actionOpen"/>
    <addaction name="actionConnect"/>
    <addaction name="actionSetting"/>
    <addaction name="actionE_it"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="actionStatus_Bar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionDelete"/>
   <addaction name="actionOpen"/>
   <addaction name="actionConnect"/>
   <addaction name="actionSetting"/>
   <addaction name="actionE_it"/>
   <addaction name="actionLed"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="MotorTesting.qrc">
     <normaloff>:/icon/icon/AddNew.png</normaloff>:/icon/icon/AddNew.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;New(Ctr+N)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="MotorTesting.qrc">
     <normaloff>:/icon/icon/Delete.png</normaloff>:/icon/icon/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete(Ctr+D)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="MotorTesting.qrc">
     <normaloff>:/icon/icon/Open.png</normaloff>:/icon/icon/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open(Ctr+O)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MotorTesting.qrc">
     <normaloff>:/icon/icon/Disconnect.png</normaloff>
     <normalon>:/icon/icon/Connect.png</normalon>:/icon/icon/Disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Connect(Ctr+C)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="MotorTesting.qrc">
     <normaloff>:/icon/icon/Settings.png</normaloff>:/icon/icon/Settings.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Setting(Ctr+S)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionE_it">
   <property name="icon">
    <iconset resource="MotorTesting.qrc">
     <normaloff>:/icon/icon/Exit.png</normaloff>:/icon/icon/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exit(Ctr+E)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="text">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionLed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MotorTesting.qrc">
     <normaloff>:/icon/icon/led_connect.png</normaloff>
     <normalon>:/icon/icon/led_disconnect.png</normalon>:/icon/icon/led_connect.png</iconset>
   </property>
   <property name="text">
    <string>Led</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;LED&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MotorTesting.qrc"/>
 </resources>
 <connections/>
</ui>
